content/howtos/setup-with-HA.md

```diff
@@ -47,6 +53,22 @@ In order to utilize the conversation agent in HomeAssistant, you will need to co
 ANY DEVICES THAT YOU SELECT TO BE EXPOSED TO THE MODEL WILL BE ADDED AS CONTEXT AND POTENTIALLY HAVE THEIR STATE CHANGED BY THE MODEL. ONLY EXPOSE DEVICES THAT YOU ARE OK WITH THE MODEL MODIFYING THE STATE OF, EVEN IF IT IS NOT WHAT YOU REQUESTED. THE MODEL MAY OCCASIONALLY HALLUCINATE AND ISSUE COMMANDS TO THE WRONG DEVICE! USE AT YOUR OWN RISK.
 {{% /notice %}}
 
-## Changing the prompt.
+## Changing the prompt
 
 Example on how to use the prompt can be seen [here](https://github.com/acon96/home-llm?tab=readme-ov-file#model).
+
+# Extended OpenAI Conversation
```

Comment:
I purposely did not mention this integration here because it does not work well using local-based LLMs. You may want to mention that here. This integration works really well